virttest/test_setup.py

```diff
@@ -868,7 +872,9 @@ def _enable_nat(self):
             self._iptables_add(rule)
 
     def _start_dhcp_server(self):
-        process.system("service dnsmasq stop")
+        if not utils_package.package_install("dnsmasq"):
```

Comment:
only need to install package if it's not installed.

Reply:
`package_install` will check it, so don't need to double check.